config/unicorn.rb

```diff
@@ -0,0 +1,16 @@
+worker_processes Integer(ENV["WEB_CONCURRENCY"] || 3)
+timeout 15
+preload_app true
+
+before_fork do |server, worker|
+   @sidekiq_pid ||= spawn("bundle exec sidekiq -c 2")
```

Comment:
The hack is not necessary anymore because Heroku now gives a free worker even on the free apps. You don't have to change this, I just wanted to make you aware.